src/main/java/mate/academy/bookapp/model/Book.java

```diff
@@ -0,0 +1,121 @@
+package mate.academy.bookapp.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    @Column(name = "cover_image")
+    private String coverImage;
+
+    public Long getId() {

```

Comment:
Annotation `@Data` (from your previous implementation) includes @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor (see the documentation). So, you don't need to generate all methods by yourself, instead you can use necessary Lombok annotations.